### ISSUE TITLE
Fehlermeldung "TH 08 ist an der offenen Schnittstelle nicht erlaubt"

### ISSUE THREAD

**Issue by topfit** [stesie/geierlein#76]
Hallo zusammen,

das Problem liegt daran, dass für die Erstellung der UStVA noch veraltete Schemata der Transferheaderversion 8 (TH08) verwenden.

Der Transferheaderversion 8 wird nicht mehr unterstützt und muss durch den neusten Transferheader (vermutlich Version 11) ersetzt werden.

Kann ich das selbst vornehmen oder muss das Projekt nochmal aktualisiert werden?

Vielen Dank für die Hilfe.

**Comment by stesie** [comment]
Ja, der TH 8 wurde abgekündigt, ... und es gibt eine neue Version, besagte 11.  Es gibt von der FinVerw auch einen Satz XSDs usf., die das abbilden.

Ehrlich gesagt wollte ich mich aber nicht mehr darum kümmern, in Anbetracht dessen, dass per Januar die ganze Schnittstelle abgekündigt wurde. Würde heißen ich stecke nochmal mehrere Stunden (die ich nicht so recht habe) in die Anpassung, bzw. erstmal die PDF-Doku der Schnittstelle durcharbeiten, was geändert werden muss...

**Comment by htgoebel** [comment]
Welche Alternativen gibt es für die Übergangszeit?

**Comment by nicmare** [comment]
scheisse

**Comment by htgoebel** [comment]
@nicmare Dann muss man halt ein oder zweimal einen UStVA über https://www.elster.de/ abgeben.
